internal/protocoltest/awsrestjson/api_op_EmptyInputAndEmptyOutput_test.go

```diff
@@ -120,7 +120,7 @@ func TestClient_EmptyInputAndEmptyOutput_awsRestjson1Deserialize(t *testing.T) {
 		"RestJsonEmptyInputAndEmptyOutput": {
 			StatusCode:    200,
 			BodyMediaType: "application/json",
-			Body:          []byte(``),
+			Body:          []byte(`{}`),
 			ExpectResult:  &EmptyInputAndEmptyOutputOutput{},
```

Comment:
might want to check if smithy core was updated to include this updated test case. @JordonPhillips mentioned that it should be.

Reply:
Correct the test case model was consumed so this regeneration picked up that change.